bazel/foreign_cc/hyperscan.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/src/fdr/teddy_runtime_common.h b/src/fdr/teddy_runtime_common.h
+index b76800e..6e587c2 100644
```

Comment:
Would be nice if the patch file documented why this patch is needed

Reply:
Ok, comment added.